planner_cspace/src/planner_3d.cpp

```diff
@@ -1827,7 +1837,7 @@ class Planner3dNode
         }
       }
 
-      cm_hyst_.clear(100);
+      clearHysteresis();
```

Comment:
Let's move this after the next line.

Reply:
Above `if (hyst)` block is also better to be in the computation time calculation.

Reply:
Done